info.php

```diff
@@ -0,0 +1,3 @@
+<?php
+	phpinfo();
+?>
```

Comment:
We want to expose this?

Reply:
Are we not supposed to? I don't know anything about PHP practices, I just saw that it was on the server so I added it to the repo.

Reply:
Oh lol it was? Generally for security we'd wanna expose as little to the public as possible. So maybe it's okay for it to be on the server for temp debugging purposes, but prob doesn't make sense for it to be in source control

Reply:
I just looked up phpinfo and I didn't realize that it exposed information about the server/stack, I just thought it was like any other source file. I'll delete it.